collector.go

```diff
@@ -50,6 +50,17 @@ func oidToList(oid string) []int {
 	return result
 }
 
+func listToOid(l []int) string {
+	result := ""
```

Comment:
I'm thinking this would be better done with a slice of strings.

```
var result []string
...
result = append(result strconv.Itoa(o))
...
return strings.Join(result, ":")
```

Reply:
Done, much cleaner.